package.json

```diff
@@ -14,45 +14,47 @@
     },
     "private": true,
     "dependencies": {
-        "@angular/animations": "~11.0.4",
-        "@angular/common": "~11.0.4",
-        "@angular/compiler": "~11.0.4",
-        "@angular/core": "~11.0.4",
-        "@angular/forms": "~11.0.4",
-        "@angular/platform-browser": "~11.0.4",
-        "@angular/platform-browser-dynamic": "~11.0.4",
-        "@angular/router": "~11.0.4",
+        "@angular/animations": "~15.0.4",
+        "@angular/common": "~15.0.4",
+        "@angular/compiler": "~15.0.4",
+        "@angular/core": "~15.0.4",
+        "@angular/forms": "~15.0.4",
+        "@angular/platform-browser": "~15.0.4",
+        "@angular/platform-browser-dynamic": "~15.0.4",
+        "@angular/router": "~15.0.4",
         "@syncfusion/ej2-angular-base": "*",
-        "@syncfusion/ej2-angular-grids": "*",
-        "@syncfusion/ej2-angular-popups": "*",
-        "@syncfusion/ej2-angular-charts": "*",
-        "@syncfusion/ej2-angular-inputs": "*",
         "@syncfusion/ej2-angular-buttons": "*",
         "@syncfusion/ej2-angular-calendars": "*",
+        "@syncfusion/ej2-angular-charts": "*",
         "@syncfusion/ej2-angular-dropdowns": "*",
+        "@syncfusion/ej2-angular-grids": "*",
+        "@syncfusion/ej2-angular-inputs": "*",
+        "@syncfusion/ej2-angular-layouts": "*",
         "@syncfusion/ej2-angular-navigations": "*",
-        "rxjs": "~6.6.0",
-        "tslib": "^2.0.0",
-        "zone.js": "~0.10.2"
+        "@syncfusion/ej2-angular-notifications": "^20.4.42",
```

Comment:
Use * for version

Reply:
Changed to * instead of Particular Version